nengo/solvers.py

```diff
@@ -42,7 +41,7 @@ def __call__(self, A, Y, rng=np.random, E=None):
         Y : (n_eval_points, dimensions) array_like
             Matrix of the target decoded values for each of the D dimensions,
             at each of the evaluation points.
-        rng : `numpy.random.RandomState`, optional (Default: `np.random`)
+        rng : `numpy.random.RandomState`, optional (Default: ``numpy.random``)
```

Comment:
Sphinx can't find the appropriate link target, so this change is fine.

Reply:
I tried this, but `numpy.random` is not something you can link to with the normal intersphinx mapping. I'm guessing it's because it isn't documented with `automodule`, but in any case I couldn't figure out a simple way to link to it without making the line super long (we do this in several other docstrings too fyi)